### ISSUE TITLE
How to add ticket/PoS receipt as report type?

### ISSUE THREAD

**Issue by zallaevan** [OCA/report-print-send#338]
## Module

base_report_to_printer

## Describe the bug

After successfully installing the module, I want to enable the "directly send to printer" option for the tickets printed by the PoS.
However, I can't find any information about the report type's "Model Name", "Action Type", or "Template Name".
How can I configure the module for it to print automatically the receipts? Or to not show the printer dialogue each time a transaction is successful at least.

## **Affected versions**:

Odoo 15

**Expected behaviour**

The module is configured correctly so after I click on Pay on the PoS screen, a ticket is automatically printed (and the drawer is opened).

**Additional context**

The module works correctly with other report types: if I click print in an invoice it automatically prints it in my printer, no issues.

**Comment by zallaevan** [comment]
Can someone mark this issue as "no stale", please? Thanks.

**Comment by an21mukhin** [comment]
POS is written on JS, so it's templates not appear in Settings/Technical/Reports or in Settings/Printing/Reports menu of this module.
Maybe you have already find the way to "convert" them to classic Reports?)

**Comment by zallaevan** [comment]
Thank you @an21mukhin for your response. Unfortunately, we have moved on and no longer use Odoo, as it is more targeted towards businesses that use CRM and such, and we needed a simpler POS system for retail. The main motivation for using Odoo was because it is open source and because it had a plugin that enabled a mandatory tax agency connection, but we have found other alternatives. Thank you again.
